tests/unit/test_network_connections.py

```diff
@@ -3461,15 +3463,15 @@ def test_interface_name_ethernet_default(self):
         """Use the profile name as interface_name for ethernet profiles"""
         cons_without_interface_name = [{"name": "eth0", "type": "ethernet"}]
         connections = ARGS_CONNECTIONS.validate(cons_without_interface_name)
-        self.assertTrue(connections[0]["interface_name"] == "eth0")
+        self.assertEqual(connections[0]["interface_name"], "eth0")
```

Comment:
https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.assertIs  this is new in Python 2.7 - in the past we needed to support Python 2.6 - is this not necessary anymore?

Reply:
https://github.com/linux-system-roles/network/blob/main/contributing.md#where-to-start needs to be updated if we don't need to support Python 2.6 anymore.

Reply:
I'm not sure if we still need to support 2.6, but if we don't, it should be a conscious, deliberate decision, in a separate PR.  So for this change, please use a method that is compatible with 2.6